src/main/java/com/mycompany/tictactoeproject/TicTacToe.java

```diff
@@ -84,11 +84,7 @@ public void run(){
     }
     
     public String returnLabel(int player){
-        if(player == 1){
-            return "x";
-        } else {
-            return "o";
-        }
+        return player == 1 ? "x" : "o";
```

Comment:
Can you use:
 return player == 1 ?  Tile.CROSS :  Tile.NOUGHT;
 instead?

Reply:
i don't know, i just write the notes